### PR TITLE
feature: change colors for invite users drawer

### PR DESCRIPTION
**Pull request by maxdiplogit**
Closes: #10700

This PR changes the dark mode colors for the Invite users drawer. Colors have been picked up as described in the issue description. The changes reflected will be as follows:

Screenshot from 2024-09-12 14-19-45

Would be happy to make any changes if required :smile:

### CODE REVIEW

File: frontend/src/ManageOrgUsers/InviteUsersForm.jsx [hunk 1]
```diff
@@ -109,7 +111,7 @@ function InviteUsersForm({
             <div className="card-header-inner-wrap">
               <h3 className="tj-text-lg tj-text font-weight-500" data-cy="add-users-card-title">
                 {!isEditing
-                  ? t('header.organization.menus.manageUsers.addNewUser', 'Add new user')
+                  ? t('header.organization.menus.manageUsers.inviteNewUsers', 'Add new user')
```
Comment (shah21) [inline frontend/src/ManageOrgUsers/InviteUsersForm.jsx:114]: Could you please revert this change? The related GitHub issue doesn't have this change requested.

**Review by shah21 — CHANGES_REQUESTED** [review]
I have added a minor comment. Could you please take a look.

**Comment by PriteshKiri** [comment]
Hey @maxdiplogit,

Are you still working on the issue?

**Comment by PriteshKiri** [comment]
Closing due to lack of progress

**Pull request closed without merging**